server/src/main/java/io/crate/auth/HttpAuthUpstreamHandler.java

```diff
@@ -149,6 +151,27 @@ private void handleHttpRequest(ChannelHandlerContext ctx, HttpRequest request) {
         }
     }
 
+    @VisibleForTesting
+    @Nullable
+    static Duration cacheControlMaxAgeFromRequest(HttpRequest request) {
+        String cacheControl = request.headers().get(HttpHeaderNames.CACHE_CONTROL.toString());
+        if (cacheControl != null) {
+            if (cacheControl.startsWith("max-age=")) {
```

Comment:
The current logic only checks if cacheControl starts with "max-age=". This could lead to potential issues if max-age is not the first directive in the Cache-Control header, but could also be a limitation of the implementation.

also see https://developer.mozilla.org/de/docs/Web/HTTP/Headers/Cache-Control

Reply:
👍